### PR TITLE
builds: merge from main into dev/4.3.x

### PR DESCRIPTION
**Pull request by LioRoger**
#### What type of PR is this?

type-build

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yhilmare — DISMISSED** [review]
lgtm

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/processor/result/DLMResultProcessor.java
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/processor/result/DLMResultProcessor.java:1]: This class is already up to date, please rollback it.

Comment (LioRoger) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/processor/result/DLMResultProcessor.java:1]: has removed

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by guowl3 — APPROVED** [review]
lgtm

**Pull request merged**